<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEinstellungen</class>
 <widget class="QDialog" name="DlgEinstellungen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="windowIcon">
   <iconset theme="settings"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollAreaEinstellungen">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaEinstellungenWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>769</width>
        <height>825</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gbEinstellungenDatenbank">
         <property name="title">
          <string>Datenbank</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Pfad</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="PushButton" name="btnDatabase">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Über dieses Feld lässt sich die gewünschte &lt;span style=&quot; font-weight:700;&quot;&gt;Datenbank&lt;/span&gt; auswählen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="tbDatabase">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbEinstellungenAnsicht">
         <property name="title">
          <string>Ansicht</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Farbschema</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="cbTheme">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Hell</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/icons/light/light.svg</normaloff>:/images/icons/light/light.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dunkel</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/icons/dark/dark.svg</normaloff>:/images/icons/dark/dark.svg</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="PushButton" name="btnRestoreView">
            <property name="text">
             <string>Zurücksetzen</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Ansicht</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbEinstellungenSonstige">
         <property name="title">
          <string>Sonstige</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="ComboBox" name="cbLanguage">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string notr="true">Deutsch</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/lang/de.svg</normaloff>:/images/lang/de.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">English</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/lang/en.svg</normaloff>:/images/lang/en.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Svenska</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/lang/se.svg</normaloff>:/images/lang/se.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Nederlands</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/lang/nl.svg</normaloff>:/images/lang/nl.svg</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Sprache</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="cbCheckUpdate">
            <property name="text">
             <string>Auf Updates beim Start prüfen</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Log Level</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="cbShowAnimations">
            <property name="text">
             <string>Animationen</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="PushButton" name="btnCheckUpdate">
            <property name="text">
             <string>Jetzt prüfen</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="cbShowTooltips">
            <property name="text">
             <string>Tooltips</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="cbConfirmClose">
            <property name="text">
             <string>Bestätigung beim Beenden</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QCheckBox" name="cbNumberFormat">
            <property name="text">
             <string>Zahlen- und Datumsformat aus Spracheinstellung (Neustart erforderlich)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbFont">
            <property name="text">
             <string>Systemschriftart</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSpinBox" name="tbLogLevel">
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbEinstellungenModule">
         <property name="title">
          <string>Module</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Module bieten erweiterte Funktionalitäten. Diese lassen sich hier jederzeit ein- oder ausschalten.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="cbModuleSchnellgaerprobe">
            <property name="text">
             <string>Schnellgärprobe</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QLabel" name="label_18">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul aktiviert die Speiseberechnung für die Karbonisierung (Nachgärung).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="cbModuleEtiketten">
            <property name="text">
             <string>Etikett</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QCheckBox" name="cbModuleLager">
            <property name="text">
             <string>Lagerverwaltung</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QCheckBox" name="cbModulePreiskalkulation">
            <property name="text">
             <string>Preiskalkulation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Damit lassen sich zeitliche Gärverläufe protokollieren (Hauptgärung, Nachgärung und Schnellgärprobe).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLabel" name="label_16">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul verwaltet anlagespezifische Kennzahlen. Die wichtigsten sind dabei Sudhausausbeute und Verdampfungsrate. Für eine genauere Berechnung sollte dieses Modul aktiviert werden.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QCheckBox" name="cbModuleSpeise">
            <property name="text">
             <string>Speiseberechnung</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <widget class="QLabel" name="label_20">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul berechnet den Preis pro Liter Bier. Voraussetzung ist die Eingabe alle Rohstoffpreise.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul ermöglicht das Erstellen von Etiketten (Label) mit automatischer Einbindung der wichtigsten Kennzahlen (Stammwürze, IBU, Farbe, ...).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Der Gärverlauf gehört zu den wichtigsten Schritten nach dem Anstellen. Das Programm trennt in Schnellgärprobe, Hauptgärung und Nachgärung.
Die graphische Auswertung lässt gute Rückschlüsse über die Gärbedingungen zu. Störungen oder Eigenarten bestimmter Hefen, die sich schon einmal &quot;schlafen legen&quot;, lassen sich so auf einen Blick identifizieren.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLabel" name="label_17">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul aktiviert die Lagerverwaltung der Rohstoffe.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QCheckBox" name="cbModuleWasseraufbereitung">
            <property name="text">
             <string>Wasseraufbereitung</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbModuleGaerverlauf">
            <property name="text">
             <string>Gärverlauf</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Hiermit wird der Anteil an Speise verrechnet, der später in Reifeprozess für die Nachägrung – entweder in Flasche oder Fass – verwendet wird.
Unter Speise versteht man einen zurückgehaltenen Teil der Würze, der vor dem Anstellen entnommen wird. Aus seinem Volumen und Zuckergehalt lässt sich die Karbonisierung berechnen.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="label_14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul ermöglicht das Bewerten von Suden.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="cbModuleBrauuebersicht">
            <property name="text">
             <string>Brauübersicht</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="cbModuleAusdruck">
            <property name="text">
             <string>Ausdruck</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_23">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul aktiviert die Schnellgärprobe für die Vorhersage des richtigen Zeitpunktes beim Grünschlauchen.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Die Schnellgärprobe ist wichtig, wenn es etwa um die Vorhersage des richtigen Zeitpunktes beim Grünschlauchen geht. Auch lässt sich damit das Verhalten unbekannter Hefen beurteilen.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Dieses Modul ist unverzichtbar, um eine möglichst exakte Vorhersage zu Sudhausausbeute, Farbe und anderer Parameter bis Brauende zu bekommen.
Gut gepflegt, entscheidet sich hiermit wie genau das Programm arbeitet. Zu Beginn sind Standardwerte eingetragen. Sie sollten aber zeitnah mit den Daten der eigenen Brauanlage ersetzt werden.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="label_15">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>In der Brauübersicht lassen sich die wichtigsten Kennzahlen in einem zeitlichen Verlauf darstellen.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QLabel" name="label_19">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dieses Modul hilft bei der Wasseraufbereitung und kalkuliert die Wasserprofile.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QCheckBox" name="cbModuleAnlagen">
            <property name="text">
             <string>Ausrüstung</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="cbModuleBewertungen">
            <property name="text">
             <string>Bewertungen</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Bei ungebrauten Suden fasst der Spickzettel alle Informationen zum einem Brauprotokoll zusammen. Dies ist ausgedruckt am Brautag eine wertvolle Hilfe.
Bei gebrauten Suden fasst die Zusammenfassung alle relevanten Informationen zusammen.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Dieses Modul ermöglicht das Ausdrucken von Brauprotokolle oder andere Zusammenfassungen.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>widgets/pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
